README.md

```
@@ -31,249 +31,785 @@
+3. Na seção **Development** (menu lateral direito), clicar em **"Create a branch"** — o GitHub gera a branch a partir de `dev` com nome no formato `{numero}-{tipo}-{descricao}` e vincula automaticamente.
+4. Mover o card da issue de **Ready** para **In Progress** no board.
+5. Fazer checkout da branch localmente:
+   ```bash
+   git fetch origin
+   git checkout 222-refactor-reformulacao-do-fluxo-de-login
+   ```
 
+#### 2. Desenvolver
 
-### 4. Configure as funções do banco de dados
+6. Realizar os commits seguindo o padrão de [Conventional Commits](#-conventional-commits).
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)